usaspending_api/references/models.py

```diff
@@ -482,8 +482,8 @@ class Meta:
         unique_together = (('program_activity_code', 'budget_year', 'responsible_agency_id', 'allocation_transfer_agency_id', 'main_account_code'),)
 
 
-class CFDAProgram(DataSourceTrackedModel):
-    program_number = models.TextField(primary_key=True, max_length=7)
+class CFDA(DataSourceTrackedModel):
```

Comment:
I'm irrationally happy that you've taken this opportunity to give this model a better name. I know it's awkward when the name is actually an acronym, but do you think there's any value in calling it Cfda for ease of typing?